### PR TITLE
[libhv] Update version

### PR DESCRIPTION
**Pull request by endingly**
updates an existing port for libhv: 

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/libhv/portfile.cmake [hunk 1]
```diff
@@ -1,11 +1,9 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ithewei/libhv
-    REF 3a67fe506a291ee94ed80bbdd73553fcb6a6b1ef #v1.3.0
-    SHA512 3aa341a94950b0d4f3351a6f62a4c34947b481bb59f1954dd8d031313e73d9c6add1017cf386c507b231b114d80cfc0739a50bed179bdbc31052c86e2f1321a3
+    REF v${VERSION} #v1.3.0
```
Comment (Cheney-W) [inline ports/libhv/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

**Comment by Cheney-W** [comment]
The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

**Comment by endingly** [comment]
@FrankXie05  I need to confirm the possible consequences of removing the patch.

**Comment by FrankXie05** [comment]
@endingly Doesn't produce any bad results, in fact it should be removed. Because upstream has merged this patch.
https://github.com/ithewei/libhv/commit/d77287e954f02706502028baadc8c87e7382301b

File: ports/libhv/portfile.cmake [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
+    SHA512 7d8f552947eb464a8dd644515b9543a7961631e427033de4e1f51f1e9d2b9ca1801553a478b0a20935b50dbc274632e38a6f0438732efed9fcbf0738738ddac5
     HEAD_REF master
-    PATCHES
-        d77287e954f02706502028baadc8c87e7382301b.patch
```
Comment (FrankXie05) [inline ports/libhv/portfile.cmake:8]: Just deleted the patch name reference, and did not delete the patch file under the `ports/libhv` folder.

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: ports/libhv/vcpkg.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "libhv",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "port-version": 1,
```
Comment (FrankXie05) [inline ports/libhv/vcpkg.json:4]:
```suggestion
```
For version updates, port-version operations:
1. Remove the `port-version`.
2. Run command `./vcpkg x-add-version libhv`

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**